Build/libHttpClient.UnitTest.142.TAEF/readme.md

```diff
@@ -1,7 +0,0 @@
-To compile the TAEF project, you must install TAEF.  Here's how:
-
-1) Install Windows Driver Kit (WDK 10), version 1607 at https://developer.microsoft.com/en-us/windows/hardware/windows-driver-kit.  This will install the TAEF MSI.
-
-2) Now run, "C:\Program Files (x86)\Windows Kits\10\Testing\Runtimes\Test Authoring and Execution Framework x64-x64_en-us.msi" which installs TAEF to "C:\Program Files (x86)\Windows Kits\10\Testing"
-
```

Comment:
this is still true isn't it?  aka taef public headers aren't installed by default

Reply:
Yes.  The readme is in the unit test folder still, which was renamed for consistency (libHttpClient.<toolset>.UnitTest instead of libHttpClient.UnitTest.<toolset>).  Not sure why codeflow is showing as a delete & a move.